index.html

```diff
@@ -11,74 +11,132 @@
 </head>
 
 <body>
-  <header class="top-container">
-    <nav class="top-nav">
+  <nav class="top-nav">
+    <div class="wrapper">
       <div class="nav-left">
-        <a href="#" class="logo">BankPL</a>
-        <p class="offer-name">Konto 360°</p>
+        <a href="#" class="site-logo">BankPL</a>
+        <span class="offer-name">Konto 360°</p>
       </div>
-      <a href="#" class="nav-icon">&#9776;</a>
+      
+      <button href="#" class="nav-icon">&#9776;</button>
+
       <ul class="main-nav">
-        <li><a href="#">O koncie</a></li>
-        <li><a href="#">Pakiet Bardzo Pomocny</a></li>
-        <li><a href="#">Lubię to polecam</a></li>
-        <li><a href="#">Jak otworzyć</a></li>
+        <li class="main-nav__item"><a href="#" class="main-nav__link">O koncie</a></li>
+        <li class="main-nav__item"><a href="#" class="main-nav__link">Pakiet Bardzo Pomocny</a></li>
+        <li class="main-nav__item"><a href="#" class="main-nav__link">Lubię to polecam</a></li>
+        <li class="main-nav__item"><a href="#" class="main-nav__link">Jak otworzyć</a></li>
       </ul>
-    </nav>
-    <div class="header-title">
-      <h1>Konto 360° pełne korzyści</h1>
-      <p>z aplikacją mobilną na każdą pogodę</p>
-      <button class="btn-top">Załóż konto 360° &gt;</button>
-    </div>
-  </header>
-  <article id="banner Top">
-    <div class="bar-one">
-      <h1>O koncie</h1>
     </div>
-    <p class="simple-place"><strong>Korzystaj z konta za 0 zł</strong> w dowolnym miejscu i czasie</p>
-    <div class="banner-one">
-      <h2>0zł</h2>
-      <p><b>za prowadzenie konta, kartę i wypłaty ze wszystkich bankomatów,</b></br> jeśli aktywnie korzystasz z konta osobistego, czyli przelewasz na nie swój dochód (min. 1000zł wpływów zewnętrznych miesięcznie) i </br> płacisz min. raz w miesiącu kartą do konta lub BLIKIEM</p>
+  </nav>
+
+  <header class="hero__container">
+    <div class="wrapper hero__content">
+      <h1 class="hero__title">Konto 360° pełne korzyści</h1>
+      <span class="hero__hint">z aplikacją mobilną na każdą pogodę</span>
+      <button class="btn btn--hero hero__btn">Załóż konto 360° &gt;</button>
     </div>
-  </article>
-  <article id="banner bottom">
-    <div class="banner-two">
-      <div class="first-col">
-        <h1>0zł</h1>
-        <p>za przelewy krajowe w złotych w Millenecie i aplikacji mobilnej</p>
-      </div>
-      <div class="second-col">
-        <h1>0zł</h1>
-        <p>za przelewy na e-mail lub telefon</p>
-      </div>
-      <div class="third-col">
-        <h1>0zł</h1>
-        <p>za realizację zleceń stałych i poleceń zapłaty</p>
+  </header>
+
+  <section id="about" class="about__container section section--about">

```

Comment:
znowu - article zamienione na section. 
To sa sekcje tej strony. czyli np. prawie cala strona moglaby byc w duzym article, podzielone na sections. 

schemat ogolny takiego layoutu jak macie to jest:
```
<section>
   <div class="section-header">  // szare tlo i 1 tytul
      <div class="wrapper"></div>
   </div> 
   <div class="section-content>  // biale tlo i tresc
        <div class="wrapper"></div>
   </div>
</section>
```

### !!! najwiekszy blad - spacje w id=""
nie mozna robic spacji w id. to nie klasa ze mozna ich miec kilka. druga sprawa ze stylowac po id tez sie nie powinno - dodaje sie je aktualnie tylko dla jsa i do kotwic (ktorych sie nie uzywa juz prawie -> uzywa sie w tym celu js :D)

no i klasy:
`section` - dla wszystkich klas - jakies ogolne paddingi i kolory
`section--about` - sekcja w wersji about :D 
`about__container` - jako baza jest about, a to jest tylko jej container. BEM wcale nie mowi ze zawsze musi byc jakis glowny element typu `.about`